### PR TITLE
Disambiguate icon on Windows for preview builds/installers to use Powershell_av_colors and make daily build use Powershell_avatar instead

### LINKED ISSUE #6317: Preview Releases (and builds?) to have a different icon to help distinguish from production version
As agreed in [PR 6301](https://github.com/PowerShell/PowerShell/pull/6301), preview releases should have a different icon to distinguish it from the production version and avoid people accidentally using the preview version.
For the daily builds I already changed it using the code [here](https://github.com/PowerShell/PowerShell/blob/master/tools/install-powershell.ps1#L162) to use the [Powershell_av_colors.ico](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_av_colors.ico) icon:
image

The only remaining icon is [Powershell_avatar.ico](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_avatar.ico):
image

Therefore I suggest that the preview should use the [Powershell_av_colors.ico](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_av_colors.ico) icon instead now and the daily builds should start using the  [Powershell_avatar.ico](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_avatar.ico) icon instead.

I think, preview builds (produced e.g. by AppVeyor) should also have a different icon and I will leave it up to the maintainers which of the 2 icons to choose or if a new icon should be used for those.

### PR DESCRIPTION
**Pull request by bergmeister**
## PR Summary

Closes #6317
Using the preview side-by-side with the RTM version is a pain/confusing due to the icons being the same.
This makes the preview build (i.e. the embedded icon in `pwsh.exe`) and MSI installer package (shortcut icon and context menu) use the [Powershell_av_colors icon](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_av_colors.ico)
image

Therefore change the icon of the daily build (which used to use the Powershell_av_colors icon) to [Powershell_avatar](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_avatar.ico)
image
This means that people like me who have a shortcut to pwsh of the daily build pinned to the taskbar, probably need to re-pin it to the taskbar to receive all changes (otherwise one will only get the change in the icon of the window)

Note: The `Test-IsPreview` method had to be moved to the build module, which is OK because the packaging module has a declared dependency on the build module in its manifest.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by bergmeister** [comment]
When testing it locally, I found that everything (pwsh.exe, context menu, taskbar menu) worked, except for the  `PowerShell_ProgramsMenuShortcut` that I still need to fix, but otherwise it is ready for review

File: tools/install-powershell.ps1 [hunk 1]
```diff
@@ -174,7 +174,7 @@ try {
         }
 
         Write-Verbose "Change icon to disambiguate it from a released installation" -Verbose
-        & "~/.rcedit/rcedit-x64.exe" "$Destination\pwsh.exe" --set-icon "$Destination\assets\Powershell_av_colors.ico"
+        & "~/.rcedit/rcedit-x64.exe" "$Destination\pwsh.exe" --set-icon "$Destination\assets\Powershell_avatar.ico"
```
Comment (TravisEz13) [inline tools/install-powershell.ps1]: why isn't this one of the same icons set during the build?

Comment (bergmeister) [reply tools/install-powershell.ps1]: This is for the daily build only, see my summary.

Comment (TravisEz13) [reply tools/install-powershell.ps1]: ok, makes sense.

**Comment by bergmeister** [comment]
@TravisEz13 The PR is now fully ready for review, I just had to test the artifacts on a clean VM (locally I had some left-overs).

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**